_posts/2022-08-05-crossplane-demo.md

```diff
@@ -0,0 +1,231 @@
+---
+layout:      posts
+background:  shortBackground
+title:       "Crossplane, a Kubernetes-powered IaC Tool"
+subtitle:    ""
+date:        2022-08-05 09:00:00
+author:      "Marty Henderson"
+description: ""
+headerImg:  "/assets/images/posts/2022-08-05-crossplane/popsicle-header.jpg"
+cross-post-link: "https://nalth.is/crossplane"
+cross-post-text: "Nalthis"
+---
+
+
+Header Image - Photo by <a href="https://unsplash.com/es/@taylorheeryphoto?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Taylor Heery</a> on <a href="https://unsplash.com/s/photos/popsicle?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Unsplash</a>
+  
+_This post originally appeared on Marty Henderson's [personal blog](https://nalth.is/crossplane)._
+
+[Crossplane](crossplane.io) is a newer member of the Infrastructure-as-Code team, with many components still yet to be built or in an alpha or beta state. Although it is new, it introduces some novel ideas of how to manage infrastructure that you deploy, especially if you are familiar with Kubernetes
```

Comment:
The Markdown URL requires a scheme, this is currently redirecting to `/2022/08/05/crossplane.io`

Reply:
Fixed